xbmc/interfaces/json-rpc/schema/notifications.json

```diff
@@ -176,6 +176,20 @@
     ],
     "returns": null
   },
+  "AudioLibrary.OnExport": {
+    "type": "notification",
+    "description": "An audio library export has finished.",
+    "params": [
+      { "name": "sender", "type": "string", "required": true },
+      { "name": "data", "type": "object", "required": false,
+        "properties": {
+          "file": { "type": "string", "required": false},
+          "failcount": { "type": "integer", "minimum": 0, "required": false}
```

Comment:
You need to provide a default value for every property that is not required with e.g. `"default": 0`. Same goes for the `file` property.